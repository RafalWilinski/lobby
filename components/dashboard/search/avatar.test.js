import React from "react";
import renderer from "react-test-renderer";
import Avatar from "./avatar";

describe("Avatar", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Avatar name="name" position="pos" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
