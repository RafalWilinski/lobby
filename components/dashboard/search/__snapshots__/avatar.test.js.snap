// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
<img
  onBlur={undefined}
  onClick={undefined}
  onFocus={undefined}
  onMouseDown={undefined}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onTouchStart={undefined}
  src={undefined}
  style={
    Object {
      "borderRadius": "50%",
      "boxShadow": "1px 2px 2px 1px #aaaaaa",
      "height": "40px",
      "marginRight": "-8px",
    }
  }
/>
`;
